lib/neo4j/spatial.rb

```diff
@@ -1,143 +1,180 @@
 module Neo4j
-  module Server
+  module Core
+    # rubocop:disable Metrics/ModuleLength
     module Spatial
+      # rubocop:enable Metrics/ModuleLength
```

Comment:
I'm happier having this disabled locally than globally, but I still really don't like disabling rubocop rules